server/server/models/categories.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, String, Integer
```

Comment:
Please add docstring to **all** modules too. It might be pretty the same as for classes declared there.